Sources/Controllers/ALKConversationViewController.swift

```diff
@@ -2443,6 +2443,13 @@ extension ALKConversationViewController: ALMQTTConversationDelegate {
               UIApplication.shared.applicationState != .background else { return false }
         return true
     }
+    
+    public func handOff(toHuman conversationId: String!) {
+        guard let accountKey = ALApplozicSettings.getZendeskSdkAccountKey(), !accountKey.isEmpty else {
+            return
+        }
+        KMZendeskChatHandler.shared.initiateZendesk(key: accountKey, conversationId: conversationId)
```

Comment:
Method is called handoff, but we are initializing. Should we change the name of this method or add a comment because it doesn't look like a normal handoff?

Reply:
moved this logic to ALKConversationViewController to handle the same thing